components/event-processor-manager/org.wso2.carbon.event.processor.manager.core/src/main/java/org/wso2/carbon/event/processor/manager/core/internal/CarbonEventManagementService.java

```diff
@@ -163,8 +163,12 @@ public void init(ConfigurationContextService configurationContextService) {
         executorService.schedule(new Runnable() {
             @Override
             public void run() {
-                log.info("Starting polling event adapters");
-                getEventReceiverManagementService().startPolling();
+                try {
+                    log.info("Starting polling event adapters");
+                    getEventReceiverManagementService().startPolling();
+                } catch (Exception e) {
+                    log.error("Unexpected error occurred", e);
```

Comment:
Improve this statement by providing more context.. "Unexpected error occurred when start polling event adapters". Just a suggestion (We need to say when exception occurred as well) .